constraint/config/crds/externaldata.gatekeeper.sh_providers.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.11.3
```

Comment:
Did you mean to downgrade this?

Reply:
It looks like that is the version indicated by the makefile:

https://github.com/open-policy-agent/frameworks/blob/2e0cb3d685752d32bfa10fa36f16a126ac68c9de/constraint/Makefile#L39C2-L39C68

Reply:
Hm, that is odd -- could we separate fixing this version drift from this PR? Aka do it as a follow up? I can take it on if you don't have cycles for that.

Reply:
yea lets address this before bumping it in GK